pwa-backend/src/test/java/io/growingabit/app/model/StudentConfirmationEmailTest.java

```diff
@@ -0,0 +1,39 @@
+package io.growingabit.app.model;
+
+import org.joda.time.DateTime;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class StudentConfirmationEmailTest {
+
+  private StudentConfirmationEmail s;
+
+  @Before
+  public void setup() {
+    this.s = new StudentConfirmationEmail("email@example.com");
+  }
+
+  @Test(expected = IllegalArgumentException.class)
+  public void invalideEmail() {
+    new StudentConfirmationEmail("email");
+  }
+
+  @Test
+  public void tsExpirationIsFuture() {
+    Assert.assertTrue(this.s.getTsExpiration() > new DateTime().getMillis());
```

Comment:
Visto che ovunque usiamo google truth, usiamola anche qui no?
Secondo me meglio fare `assertThat(this.s.getTsExpiration()).isGreaterThan(new DateTime().getMillis()`